cms/static/js/models/xblock_info.js

```diff
@@ -256,19 +256,6 @@ function(Backbone, _, str, ModuleUtils) {
         */
         isEditableOnCourseOutline: function() {
             return this.isSequential() || this.isChapter() || this.isVertical();
-        },
-
-       /*
-        * Check whether any verification checkpoints are defined in the course.
-        * Verification checkpoints are defined if there exists a user partition
-        * that uses the verification partition scheme.
-        */
-        hasVerifiedCheckpoints: function() {
```

Comment:
Look @efischer19, I found more verification user partition changes I can delete! I'm removing some of the Studio code as part of my current work because I stumbled across it, and some of it I would need to modify if it stays in.